projects/plugins/jetpack/_inc/client/components/owner-disconnect-dialog/index.jsx

```diff
@@ -0,0 +1,192 @@
+/**
+ * External dependencies
+ */
+import jetpackAnalytics from '@automattic/jetpack-analytics';
+import restApi from '@automattic/jetpack-api';
+import { getRedirectUrl } from '@automattic/jetpack-components';
+import { ExternalLink, Modal, Button } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Icon, chevronRight, external } from '@wordpress/icons';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import React, { useCallback, useState, useEffect } from 'react';
+
+const ActionCard = ( {
+	title,
+	onClick = () => null,
+	isExternal = false,
+	link = '#',
+	action,
+	disabled = false,
+} ) => {
+	const disabledCallback = useCallback( e => e.preventDefault(), [] );
+
+	return (
+		<div
+			className={
+				'jp-connection__manage-dialog__action-card card' + ( disabled ? ' disabled' : '' )
+			}
+		>
+			<div className="jp-connection__manage-dialog__action-card__card-content">
+				<a
+					href={ link }
+					className={ clsx( 'jp-connection__manage-dialog__action-card__card-headline', action ) }
+					onClick={ ! disabled ? onClick : disabledCallback }
+					target={ isExternal ? '_blank' : '_self' }
+					rel={ 'noopener noreferrer' }
+				>
+					{ title }
+					<Icon
+						icon={ isExternal ? external : chevronRight }
+						className="jp-connection__manage-dialog__action-card__icon"
+					/>
+				</a>
+			</div>
+		</div>
+	);
+};
+
+const OwnerDisconnectDialog = ( { isOpen, onClose, apiRoot, apiNonce, onDisconnected } ) => {
```

Comment:
It appears that the component needs the same CSS layout fix we used for My Jetpack:
```scss
.jp-connection__disconnect-dialog .components-modal__content > div:not(.components-modal__header) {
    display: flex;
    flex-direction: column;
    height: 100%;
}
```

Reply:
Disregard that, it was my local JS building problem, works fine on JN.